js/index.js

```diff
@@ -0,0 +1,17 @@
+const today = new Date();
+const thisYear = today.getFullYear();
+const footer = document.querySelector('footer');
+const copyRight = document.createElement('p');
+
+copyRight.innerHTML = `Raíssa Gomes; ${thisYear}`;
```

Comment:
Looks good! If you want, you can use the text you have currently in the HTML here, so that you get the copyright symbol :smile: